lib/modules/manager/gradle/extract.spec.ts

```diff
@@ -605,4 +611,191 @@ describe('modules/manager/gradle/extract', () => {
       },
     ]);
   });
+
+  it('loads further scripts using apply from statements', async () => {
+    const buildFile = `
+      buildscript {
+          repositories {
+              mavenCentral()
+          }
+
+          apply from: "\${someDir}/libs1.gradle"
+          apply from: file("gradle/libs2.gradle")
+          apply from: "gradle/libs3.gradle"
+          apply from: file("gradle/non-existing.gradle")
+
+          dependencies {
+              classpath "com.google.protobuf:protobuf-java:\${protoBufVersion}"
+              classpath "com.google.guava:guava:\${guavaVersion}"
+              classpath "io.jsonwebtoken:jjwt-api:0.11.2"
+
+              classpath "org.junit.jupiter:junit-jupiter-api:\${junitVersion}"
+              classpath "org.junit.jupiter:junit-jupiter-engine:\${junitVersion}"
+          }
+      }
+    `;
+
+    mockFs({
```

Comment:
@rarkins @HonkingGoose we should add a section about filesystem mocking to our dev best practises.

We've a lot different codes. i would prefer our `Fixtures` class.

Reply:
Let's discuss the requirements in:

- https://github.com/renovatebot/renovate/issues/16319